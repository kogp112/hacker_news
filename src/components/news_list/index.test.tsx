import * as React from "react";
import * as renderer from "react-test-renderer";
import NewsList from "./index";

let container = null;

beforeEach(() => {
  container = document.createElement("div");
  document.body.appendChild(container);
});

afterEach(() => {
  container = null;
});

it("should render news list", () => {
  const newsId = "";
  const index = 0;

  const component = renderer
  .create(<NewsList newsId={newsId} index={index} />)
  .toJSON();
  expect(component).toMatchSnapshot();
});