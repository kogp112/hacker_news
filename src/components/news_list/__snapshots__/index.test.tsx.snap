// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render news list 1`] = `
<div
  className="row"
  onClick={[Function]}
>
  <p
    className="index"
  >
     
  </p>
  <p
    className="newstitle"
  >
    0
      
  </p>
</div>
`;
